### PR TITLE
Get basic information by keywords

### PR DESCRIPTION
**Pull request by levent91**
Added simplified results for keywords search.

@zpelechova let's test a bit.

### CODE REVIEW

File: src/crawler_utils.js [hunk 1]
```diff
@@ -210,10 +176,13 @@ exports.handleDetail = async (page, request, extendOutputFunction, subtitlesSett
  * @param {number} maxRequested
  * @param {boolean} isSearchResultPage
  * @param {object} input
+ * @param {string} requestUrl
  */
 
-const getBasicInformation = async (page, maxRequested, isSearchResultPage, input) => {
-    const { youtubeVideosSection, youtubeVideosRenderer, url, videoTitle, channelNameText, subscriberCount, canonicalUrl } = CONSTS.SELECTORS.SEARCH;
+const getBasicInformation = async (page, maxRequested, isSearchResultPage, input, requestUrl) => {
```
Comment (petrpatek) [inline src/crawler_utils.js:182]: Five parameters are quite a lot, and the order can be easily mixed up. I would either wrap the parameters in some context object so the order does not matter or refactor it into multiple more straightforward functions.

File: src/crawler_utils.js [hunk 2]
```diff
@@ -224,103 +193,200 @@ const getBasicInformation = async (page, maxRequested, isSearchResultPage, input
...
+                            duration,
+                        });
+
+                        if (videoAmount >= maxRequested) {
+                            shouldContinue = false;
+                            break;
+                        }
 
-                    videoCount++;
+                        await sleep(CONSTS.DELAY.HUMAN_PAUSE.max);
```
Comment (petrpatek) [inline src/crawler_utils.js:263]: It feels a bit weird that the last property is not uppercase.

**Review by petrpatek — CHANGES_REQUESTED** [review]
Great job! I have just a two small nitpicks.

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]

File: src/crawler_utils.js [hunk 3]
```diff
@@ -224,103 +220,207 @@ const getBasicInformation = async (page, maxRequested, isSearchResultPage, input
...
+        log.debug('loadVideosUrls', { maxRequested });
+        let shouldContinue = true;
+        let videoAmount = 0;
+
+        const logInterval = setInterval(
+            () => log.info(`Scrolling state - Pushed ${videoAmount} unique videos total`),
+            60000,
+        );
+
+        try {
```
[the remark this answers is not in the document]
Comment (levent91) [reply src/crawler_utils.js]: tried to clean it up a bit, still not satisfied but looks better. Also added "/user/videos" type of channel format too, so let's try this one instead.

[the remark this answers is not in the document]
Comment (metalwarrior665) [reply src/crawler_utils.js]: Looks definitely better

**Comment by metalwarrior665** [comment]
@levent91 Ready to merge now?

**Comment by levent91** [comment]
Yes, we can test now

**Pull request merged**